--- a/examples/hotrod/cmd/root.go
+++ b/examples/hotrod/cmd/root.go
@@ -98,12 +98,9 @@ func onInitialize() {
 		zap.AddStacktrace(zapcore.FatalLevel),
 		zap.AddCallerSkip(1),
 	}
-	if !verbose {
-		zapOptions = append(zapOptions,
-			zap.IncreaseLevel(zap.LevelEnablerFunc(func(l zapcore.Level) bool { return l != zapcore.DebugLevel })),
-		)
-	}
-	logger, _ = zap.NewDevelopment(zapOptions...)
+	loggerConfig := zap.NewProductionConfig()
+	loggerConfig.Level.SetLevel(zapcore.ErrorLevel)
+	logger, _ = loggerConfig.Build(zapOptions...)

 	jaegerclientenv2otel.MapJaegerToOtelEnvVars(logger)

